apps/ledger-live-mobile/package.json

```diff
@@ -253,7 +256,6 @@
     "flow-bin": "0.122.0",
     "flow-typed": "^2.6.2",
     "jest": "^28.1.1",
-    "jest-circus": "^28.1.1",
```

Comment:
@nabil-brn @abdurrahman-ledger Can you remember if it was you that removed jest-circus and if so why? We are still using it in the `apps/ledger-live-mobile/e2e/config.json`